site/src/components/Resources/ResourceCard.tsx

```diff
@@ -149,7 +149,13 @@ export const ResourceCard: FC<ResourceCardProps> = ({
                   </div>
                 </Stack>
 
-                <Stack direction="row" alignItems="center" spacing={0.5}>
+                <Stack
+                  direction="row"
+                  alignItems="center"
+                  spacing={0.5}
+                  wrap="wrap"
+                  maxWidth="750px"
```

Comment:
Is `maxWidth` to prevent the left column from getting squashed?  If so I wonder if we could do something like put `white-space: nowrap` or similar on the left column instead then the right can take up whatever space is left over.  For example with the current `max-width` setting you can still get some wonkiness depending on your screen size:

![width](https://user-images.githubusercontent.com/45609798/197270231-d3c84e5c-2ddf-4e8c-b106-0f34244aa3e0.png)

With no-wrap:

![nowrap](https://user-images.githubusercontent.com/45609798/197270320-d9473adb-6490-4b5b-82d1-9f9d44e9c4f9.png)